README.md

```diff
@@ -85,4 +85,7 @@ Now visit [`http://localhost:5000/`](http://localhost:5100/) in your web browser
 docker-compose up --build 
 ```
 
-Now visit [`http://localhost:8080/`](http://localhost:8080/) in your web browser to view the app.
\ No newline at end of file
+Now visit [`http://localhost:8080/`](http://localhost:8080/) in your web browser to view the app.
+
+## Viewing the Architecture Diagrams 
+ Visit https://app.diagrams.net/ and select "Open Exsisting Diagram" option, navigate to “documentation” folder and select the appropriate diagram to view
```

Comment:
Minor, but it would be nice to make that a link:
```suggestion
 Visit [https://app.diagrams.net/](https://app.diagrams.net/) and select "Open Exsisting Diagram" option, navigate to “documentation” folder and select the appropriate diagram to view
```